src/timeseriesflattener/feature_cache/cache_to_disk.py

```diff
@@ -0,0 +1,185 @@
+"""Cache module for writing features to disk."""
+import datetime as dt
+import os
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+
+from timeseriesflattener.feature_cache.abstract_feature_cache import FeatureCache
+from timeseriesflattener.feature_spec_objects import TemporalSpec
+from timeseriesflattener.utils import load_dataset_from_file, write_df_to_file
+
+
+class DiskCache(FeatureCache):
+    """Cache module for writing features to disk."""
+
+    def __init__(
+        self,
+        feature_cache_dir: Path,
```

Comment:
I would recommend a default - but we already talked about it - so if you find it is not ideal feel free to ignore this comment